x/committee/types/committee.go

```diff
@@ -211,23 +128,40 @@ func (c BaseCommittee) HasPermissionsFor(ctx sdk.Context, appCdc *codec.Codec, p
 func (c BaseCommittee) GetVoteThreshold() sdk.Dec { return c.VoteThreshold }
 
 // SetVoteThreshold is a setter for committee VoteThreshold
-func (c BaseCommittee) SetVoteThreshold(voteThreshold sdk.Dec) BaseCommittee {
+func (c *BaseCommittee) SetVoteThreshold(voteThreshold sdk.Dec) {
 	c.VoteThreshold = voteThreshold
-	return c
 }
 
 // GetProposalDuration is a getter for committee ProposalDuration
 func (c BaseCommittee) GetProposalDuration() time.Duration { return c.ProposalDuration }
 
 // SetProposalDuration is a setter for committee ProposalDuration
-func (c BaseCommittee) SetProposalDuration(proposalDuration time.Duration) BaseCommittee {
+func (c *BaseCommittee) SetProposalDuration(proposalDuration time.Duration) {
 	c.ProposalDuration = proposalDuration
-	return c
```

Comment:
@denalimarsh do you remember why we don't set this directly on the committee? i noticed we are using these in simulations but the return values were never used.